completions/cvs

```diff
@@ -1,5 +1,10 @@
 # cvs(1) completion                                        -*- shell-script -*-
 
+_comp_xfunc_cvs_roots()
+{
+    _cvs_roots
+}
+
```

Comment:
Should we not rename + `_comp_deprecate_func` `_cvs_roots` to this instead?

Reply:
I have kept the original function `_cvs_roots` for internal use by `cvs` and just added an interface here because there seem to be many other functions similar to `_cvs_roots` below (i.e., _cvs_entries, _cvs_modules, _cvs_commands. etc.)

- Nevertheless, I may directly rename only `_cvs_roots` in the subsequent code.
- Another option is to make all the similar functions *xfunc* for consistency.

There are similar cases also in several other files.

Reply:
Hmm, I see. But then again all cases except this and ssh in this PR take the rename+deprecate approach. None of the choices strikes me obviously better than all others, but I think I'd prefer the consistency by not leaving wrappers like this behind, and making things xfunc only when we know we have xfunc use for them from somewhere else.

Reply:
OK! So, if I don't misunderstand, ccf13887?

Reply:
Yes, that's what I meant, thanks.